config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_alloydbinstances.alloydb.cnrm.cloud.google.com.yaml

```diff
@@ -242,6 +263,11 @@ spec:
                   current reported status reflects the most recent desired state of
                   the resource.
                 type: integer
+              publicIpAddress:
```

Comment:
ditto publicIPAddress

Reply:
The other suggestion is about an output-only field under `spec`. This output-only field is under `status` so it is a different situation.

But @nancynh, if this is a field you don't expect users to leverage and @yuwenma if you are feeling strongly about supporting the acronym then we can just ignore this field for now.

Reply:
Ah okay, thanks for the explanation! I expect users will want to use this field to know which public IP address they can use to connect to the DB